test/simplify_qdq_test.cpp

```diff
@@ -0,0 +1,636 @@
+        auto d6 = add_quantize_op(m2, "dequantizelinear", c1, sc1, z1);
+        m2.add_return({d6});
+    }
+
+    run_pass(m1);
+    EXPECT(m1 == m2);
+}
+
+TEST_CASE(conv_multi_scale)
+{
+    migraphx::shape s1{migraphx::shape::int8_type, {1}};
+    migraphx::shape s4{migraphx::shape::int8_type, {1280, 320, 1, 1}};
+    migraphx::shape s7{migraphx::shape::float_type, {1, 320, 7, 7}};
+    migraphx::shape s8{migraphx::shape::float_type, {320}};
+
+    std::vector<int> zero{0};
+
+    migraphx::module m1;
+    {
+        auto input   = m1.add_parameter("input", s7);
+        auto weights = m1.add_parameter("weights", s4);
+        auto l1      = m1.add_literal(s1, zero);
+        auto l2      = m1.add_literal(migraphx::generate_literal(s8, 0));
+
+        auto d1 = add_quantize_op(m1, "dequantizelinear", weights, l2, l1);
+        auto q1 = add_quantize_op(m1, "quantizelinear", input, l2, l1);
+        auto d5 = add_quantize_op(m1, "dequantizelinear", q1, l2, l1);
+        auto c1 = m1.add_instruction(migraphx::make_op("convolution",
+                                                       {{"padding", {0, 0, 0, 0}},
+                                                        {"stride", {1, 1}},
+                                                        {"dilation", {1, 1}},
+                                                        {"group", 1},
+                                                        {"padding_mode", 0}}),
+                                     d5,
+                                     d1);
+        m1.add_return({c1});
+    }
+
+    migraphx::module m2;
+    {
+        auto input   = m2.add_parameter("input", s7);
+        auto weights = m2.add_parameter("weights", s4);
+        auto l1      = m2.add_literal(s1, zero);
+        auto l2      = m2.add_literal(migraphx::generate_literal(s8, 0));
+
+        auto d1 = add_quantize_op(m2, "dequantizelinear", weights, l2, l1);
+        auto c1 = m2.add_instruction(migraphx::make_op("convolution",
+                                                       {{"padding", {0, 0, 0, 0}},
+                                                        {"stride", {1, 1}},
+                                                        {"dilation", {1, 1}},
+                                                        {"group", 1},
+                                                        {"padding_mode", 0}}),
+                                     input,
+                                     d1);
+        m2.add_return({c1});
+    }
+
+    run_pass(m1);
+    EXPECT(m1 == m2);
+}
+
+TEST_CASE(conv_bias_add)
+{
+    migraphx::shape s1{migraphx::shape::int8_type, {1}};
+    migraphx::shape s4{migraphx::shape::int8_type, {1280, 320, 1, 1}};
+    migraphx::shape s6{migraphx::shape::int32_type, {1280}};
+    migraphx::shape s7{migraphx::shape::float_type, {1, 320, 7, 7}};
+    migraphx::shape s8{migraphx::shape::float_type, {1}};
+
+    std::vector<float> scale{0.5};
+    std::vector<int> zero{0};
```

Comment:
There should be no need for these extra vector classes as you can construct literal scalars with `add_literal(value)`.